### ISSUE TITLE
ERR! code ELIFECYCLE with `npm start` on Windows 7

### ISSUE THREAD

**Issue by mattbk** [badges/shields#1259]
Reticketed from https://github.com/badges/shields/issues/653#issuecomment-343233556.

I assumed that this was a problem with my node/npm install, because it works on Mac OS for me. I don't have much experience with node.js.

When running `npm start' on Windows 7, I get:
```
> gh-badges@1.3.0 start C:\Users\mattbk\Documents\GitHub\shields
> INFOSITE=/ node server 8080 ::

'INFOSITE' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gh-badges@1.3.0 start: `INFOSITE=/ node server 8080 ::`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gh-badges@1.3.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\mattbk\AppData\Roaming\npm-cache\_logs\2017-11-09T19_36_33_044Z-debug.log
```
That log file shows:
```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Users\\mattbk\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'start' ]
2 info using npm@5.5.1
3 info using node@v8.9.1
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle gh-badges@1.3.0~prestart: gh-badges@1.3.0
6 info lifecycle gh-badges@1.3.0~start: gh-badges@1.3.0
7 verbose lifecycle gh-badges@1.3.0~start: unsafe-perm in lifecycle true
8 verbose lifecycle gh-badges@1.3.0~start: PATH: C:\Users\mattbk\AppData\Roaming\npm\node_modules\npm\bin\node-gyp-bin;C:\Users\mattbk\Documents\GitHub\shields\node_modules\.bin;C:\Program Files\cmder\vendor\conemu-maximus5\ConEmu\Scripts;C:\RBuildTools\3.4\bin;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\CMG\CMGJobService;C:\RBuildTools\3.4\mingw_32\bin;C:\Program Files\ImageMagick-6.9.3-Q16;C:\Program Files\Microsoft MPI\Bin\;C:\Program Files\Rtools\bin;C:\Program Files\Rtools\gcc-4.6.3\bin;C:\Program Files (x86)\CMG\CMGJobService;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files (x86)\Hewlett-Packard\HP Performance Advisor;C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\;C:\Program Files (x86)\MySQL\MySQL Utilities 1.4.3\;C:\Program Files (x86)\MySQL\MySQL Utilities 1.4.3\Doctrine extensions for PHP\;C:\Program Files\Microsoft DNX\Dnvm\;C:\Program Files\cURL\bin;C:\Program Files (x86)\QuickTime\QTSystem\;C:\ProgramData\chocolatey\bin;C:\Python27\ArcGIS10.4;C:\Python27\ArcGIS10.4\Scripts;C:\Program Files\R\R-3.3.1\bin\;C:\Python36miniconda\;C:\Python36miniconda\Scripts;C:\Python36miniconda\Library\bin;C:\HashiCorp\Vagrant\bin;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\120\Tools\Binn\;C:\Program Files\Git\cmd;C:\Program Files\nodejs\;C:\Ruby24-x64\bin;C:\Python36\Scripts\;C:\Python36\;C:\Program Files\cmder\bin;C:\Program Files\Git\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\share\vim\vim74;C:\Program Files\cmder\vendor\conemu-maximus5;C:\Program Files\cmder\vendor\conemu-maximus5\ConEmu;C:\Program Files\ImageMagick-6.9.3-Q16;C:\Program Files\Microsoft MPI\Bin\;C:\ProgramData\Oracle\Java\javapath;C:\Program Files\Rtools\bin;C:\Program Files\Rtools\gcc-4.6.3\bin;C:\Program Files (x86)\CMG\CMGJobService;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\ATI Technologies\ATI.ACE\Core-Static;C:\Program Files (x86)\Hewlett-Packard\HP Performance Advisor;C:\Program Files (x86)\MiKTeX 2.9\miktex\bin\;C:\Program Files (x86)\MySQL\MySQL Utilities 1.4.3\;C:\Program Files (x86)\MySQL\MySQL Utilities 1.4.3\Doctrine extensions for PHP\;C:\Python27\;C:\Users\mattbk\.dnx\bin;C:\Program Files\Microsoft DNX\Dnvm\;C:\Program Files\cURL\bin;C:\Program Files (x86)\QuickTime\QTSystem\;C:\Program Files\Git\cmd;C:\ProgramData;C:\Python27\Scripts;C:\Program Files\PostgreSQL\9.5\bin\pg_config.exe;"C:\Program Files\R\R-3.3.1\bin\x64\R.exe";"C:\Program Files\R\R-3.3.1\";C:\Users\mattbk\AppData\Roaming\npm
9 verbose lifecycle gh-badges@1.3.0~start: CWD: C:\Users\mattbk\Documents\GitHub\shields
10 silly lifecycle gh-badges@1.3.0~start: Args: [ '/d /s /c', 'INFOSITE=/ node server 8080 ::' ]
11 silly lifecycle gh-badges@1.3.0~start: Returned: code: 1  signal: null
12 info lifecycle gh-badges@1.3.0~start: Failed to exec start script
13 verbose stack Error: gh-badges@1.3.0 start: `INFOSITE=/ node server 8080 ::`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Users\mattbk\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\index.js:280:16)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at EventEmitter.emit (events.js:214:7)
13 verbose stack     at ChildProcess.<anonymous> (C:\Users\mattbk\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at ChildProcess.emit (events.js:214:7)
13 verbose stack     at maybeClose (internal/child_process.js:925:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
14 verbose pkgid gh-badges@1.3.0
15 verbose cwd C:\Users\mattbk\Documents\GitHub\shields
16 verbose Windows_NT 6.1.7601
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\mattbk\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "start"
18 verbose node v8.9.1
19 verbose npm  v5.5.1
20 error code ELIFECYCLE
21 error errno 1
22 error gh-badges@1.3.0 start: `INFOSITE=/ node server 8080 ::`
22 error Exit status 1
23 error Failed at the gh-badges@1.3.0 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```
I can start the server directly with `node server.js`, however.

**Comment by paulmelnikow** [comment]
Hi, thanks for that information. Proposed fix in #1260. Could you try running it and seeing if it fixes the problem? Thanks!

**Comment by mattbk** [comment]
Can confirm that the fix in #1259 works for me on Windows 7. :+1:
